test/runner.js

```diff
@@ -298,6 +298,25 @@ function testAPI(esprima, code, expected) {
     }
 }
 
+function generateTestCase(esprima, testCase) {
+    var tree, fileName = testCase.key + ".tree.json";
+    try {
+        tree = esprima.parse(testCase.case, {loc: true, range: true});
```

Comment:
That can be a subsequent improvement, e.g. combining it with the generated tokens for the existing tests (see #1027).